d2s.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": "Pedro Araujo",
+            "oem": "",
+            "device": "d2s",
+            "filename": "crDroidAndroid-15.0-20250120-d2s-v11.1.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/d2s/11.x/crDroidAndroid-15.0-20250120-d2s-v11.1.zip/download",
+            "timestamp": 1737427864,
+            "md5": "059ae3e7a4ea4808a174acd191c13225",
+            "sha256": "46f80e4e41640315caf74645dacc1712e7442d66d22104c4267756279072382d",
+            "size": 1390598202,
+            "version": "11.1",
+            "buildtype": "Monthly",
+            "forum": "https://xdaforums.com/t/rom-official-15-crdroid-11-1-aosp-encryption.4714988/",
```

Comment:
correct forum links
in order to comply with XDA posting rules, don't include telegram urls
add it in the device.json file and will show up on support page